stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -394,8 +394,8 @@ spec:
           mountPath: {{ .Values.artifactory.customPersistentPodVolumeClaim.mountPath }}
         {{- end }}
       {{- if .Values.artifactory.userPluginSecrets }}
-        - name: tmp-plugins
-          mountPath: "/tmp/plugins/"
+        - name: bootstrap-plugins
```

Comment:
@kpirateone Not sure this is needed in member nodes ?  can you please verify this - https://www.jfrog.com/confluence/display/JFROG/User+Plugins#UserPlugins-DeployingPlugins

`Artifactory HA Plugins Directory`

`If you are working with a High Availability cluster your user plugins should be added to the primary node and they will be propagated to the entire cluster.
`
@eldada  ^

Reply:
@chukka Those lines were there so i modified them to be consistent.  I am not fully sure i have the contextual knowledge or capacity to test whether or not this can be removed on how plugins have change internally to say 'userpluginsecrets' are no longer needed in the nodes.  maybe if they are not needed, the can be removed in a separate PR?

Reply:
@kpirateone we would like to the test this PR changes and come back with results. Till then I would advise this PR on hold (for merging) ?

Reply:
@chukka of course, thanks for your diligence